packages/app/src/main.ts

```diff
@@ -30,7 +30,7 @@ app.use(Toast, {
   position: POSITION.BOTTOM_RIGHT,
 })
 
-app.use(urql, makeUrqlClient({ target: 'app', socketIoRoute: config.socketIoRoute }))
+app.use(urql, makeUrqlClient({ target: 'app', namespace: config.namespace, socketIoRoute: config.socketIoRoute }))
```

Comment:
Could we change line 23 to:

const { namespace, socketIoRoute } = JSON.parse(decodeBase64Unicode(window.__CYPRESS_CONFIG__.base64Config)) as Cypress.Config

Reply:
Yeah, let's just do that in another PR with other changes so we don't need to wait on CI